packages/smart-camera-web/smart-camera-web.js

```diff
@@ -1898,9 +1898,26 @@ class SmartCameraWeb extends HTMLElement {
 
   _capturePOLPhoto() {
     const canvas = document.createElement('canvas');
-    canvas.width = 240;
-    canvas.height =
-      (canvas.width * this._video.videoHeight) / this._video.videoWidth;
+
+    // Determine orientation of the video
+    const isPortrait = this._video.videoHeight > this._video.videoWidth;
+
+    // Set dimensions based on orientation, ensuring minimums
+    if (isPortrait) {
+      // Portrait orientation (taller than wide)
+      canvas.width = 240;
+      canvas.height = Math.max(
+        320,
+        (canvas.width * this._video.videoHeight) / this._video.videoWidth,
```

Comment:
I think you should wrap these calculations in a Math.min so we can enforce reasonable maximums as well. Something like:
```suggestion
        Math.min((canvas.width * this._video.videoHeight) / this._video.videoWidth, <resonableMaximum>),
```
I would do this for all the calculated values

Reply:
Also we perform this same calculation in a ton of places. It would make a lot of sense to have this in a function we can share so we have a calulatePortraitHeight function and a calculateLandscapeWidth function.

Reply:
Hmm, that's a good point. but from the story, it seems we're only having issues with the images being too small. @okufoniyi have we had any issues with selfie captures being too large in only one dimension? What do we feel is a reasonable maximum if we were to enforce one?

The highest aspect ratio I've encountered for a device in the real world is 2:1. Even then, I doubt that their cameras capture in that aspect ratio. But if we assume that to be the maximum aspect ratio for example, then capturing a selfie image at 480px on the smaller dimension would put the bigger dimension at 960px. Do we want to set 960px as the maximum for the bigger dimension for selfies? and 480px for liveness images?

Reply:
That sounds reasonable to me, but I defer to Ope and the CVML team on what a reasonable maximum would value.

Reply:
I don’t think we’ve run into any issues with the selfie capture being too large yet. That said, I get how this change could cause that, so Chris makes a good point. My only concern is that setting a max value might mess with the aspect ratio and skew the image, which could bring in a whole new set of problems.

I’m thinking it might be better to handle the max value as a separate story because of that but I’d love to get everyone’s thoughts. Let’s chat about it after stand-up tomorrow.

Reply:
Following our chat, we will tackle enforcing maximums as a separate story after discussing with mobile. I'll go ahead & LGTM this now. @beaglebets @khalibloo.